### DISCUSSION TITLE
NAV mode and displayed flightpath on ND

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#7225] [General]
Hello dev team,

today, after newest dev version update the refreshing of the flightpath on the ND is not working proberly. You can see it on the FO side. At the cpt. side I did a workaround with changing from ARC mode to NAV and back on the EFIS. Then it is refreshed ...

![grafik](https://user-images.githubusercontent.com/76531893/169482451-5832cdc9-9933-4e3d-80cb-a22c0df1ad51.png)

EDIT: Was one hour out of cockpit, came back and AP wasn´t in NAV mode anymore.AP was in HDG mode and far away from the flightplan path. Tried my workaround like described above ... CTD ....

Seems there might be a general issue with NAV mode  .... hmmm ..

greets Klaus

**Comment by tracernz** [comment]
Hi Klaus,
I beleive this was a short-term bug due to some performance improvement work. It should have been resolved some days ago.
